rskj-core/src/test/java/co/rsk/peg/federation/VoteFederationChangeTest.java

```diff
@@ -430,6 +430,42 @@ void voteCommitFederation_postRSKIP186_preRSKIP419_whenPendingFederationIsSet_sh
         assertLogCommitFederation(oldFederation, newFederation);
     }
 
+    @Test
+    void voteCommitFederation_postRSKIP419_preRSKIP305_whenPendingFederationIsSet_shouldPerformCommitFederationActions() {
```

Comment:
```suggestion
    void voteCommitFederation_postRSKIP419_preRSKIP305_whenPendingFederationIsSet_shouldPerformCommitFederationActionsAndBuildP2shErpFed() {
```

lets add that to the name